config/routes.rb

```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  resources :greetings, only: [:index]
```

Comment:
- [**Optional**] As a general rule of thumb, let's always version our API. Versioning allows developers to improve their API in the future without breaking the client's application when new updates are available.
- Hint

```.ruby
 namespace :api do
    namespace :v1 do
      resources :greetings, only: [:index], defaults: { format: 'json' }
    end
  end
```

You'll also need to modify the controller path api/V1